xbmc/settings/Settings.cpp

```diff
@@ -658,7 +658,7 @@ void CSettings::InitializeDefaults()
 #endif // defined(TARGET_POSIX)
 
 #if defined(TARGET_WINDOWS)
-  std::static_pointer_cast<CSettingString>(GetSettingsManager()->GetSetting(CSettings::SETTING_MUSICPLAYER_VISUALISATION))->SetDefault("visualization.milkdrop");
+  std::static_pointer_cast<CSettingString>(GetSettingsManager()->GetSetting(CSettings::SETTING_MUSICPLAYER_VISUALISATION))->SetDefault("");
```

Comment:
Can't you just remove the entire line?

Reply:
No because the default musicplayer visualization is set to `visualization.spectrum` for all platforms. So if you want no default visualization on win32 you need to specify that. The question is whether we should actually do that in the platform-specific settings XML instead of doing it here. I'm not sure why I did it this way.

Reply:
imo the platform xml makes most sense. I guess this is just a left over

Reply:
I agree with Martijn, we have platform xml. remove this line and define defaults in windows.xml